src/app/settings/SettingsPage.tsx

```diff
@@ -198,275 +199,290 @@ class SettingsPage extends React.Component<Props> {
       }
     );
 
+    const menuItems = _.compact([
```

Comment:
Does compact here remove items that are not translated (undefined)?

Reply:
No, it's left over from the progress page where certain items could be undefined. Everything is translated.